tests/model_registry/utils.py

```diff
@@ -156,3 +161,71 @@ def get_model_registry_db_label_dict(db_resource_name: str) -> dict[str, str]:
         Annotations.KubernetesIo.INSTANCE: db_resource_name,
         Annotations.KubernetesIo.PART_OF: db_resource_name,
     }
+
+
+def get_pod_container_error_status(pod: Pod) -> str:
```

Comment:
should be renamed, returns `waiting`  reason. also , please move the comment to a docstring, so a user will not have to check a comment in the code